src/node/panner.rs

```diff
@@ -271,16 +282,14 @@ pub struct PannerNode {
     orientation_x: AudioParam,
     orientation_y: AudioParam,
     orientation_z: AudioParam,
-    cone_inner_angle: Arc<AtomicF64>,
-    cone_outer_angle: Arc<AtomicF64>,
-    cone_outer_gain: Arc<AtomicF64>,
-    distance_model: Arc<AtomicU8>,
-    ref_distance: Arc<AtomicF64>,
-    max_distance: Arc<AtomicF64>,
-    rolloff_factor: Arc<AtomicF64>,
+    cone_inner_angle: AtomicF64,
```

Comment:
Why still individual atomics instead of an inner typed wrapped into a single `RefCell` as demonstrated in in the `AudioBufferSourceNode`? This also applies to all other node types as far as I could see.